peerdiversity/filter.go

```diff
@@ -220,6 +220,13 @@ func (f *Filter) ipGroupKey(ip net.IP) (PeerIPGroupKey, error) {
 		if err != nil {
 			return "", fmt.Errorf("failed to fetch ASN for IPv6 addr %s: %w", ip.String(), err)
 		}
+
+		// if no ASN found then fallback on using the /32
+		if len(s) == 0 {
+			dfLog.Debugw("group key is empty", "appKey", f.logKey, "ip", ip.String())
+			s = net.CIDRMask(32, 128).String()
```

Comment:
can we prefix `s` in this case so that it map into the same space as ASNs? (i'm not sure why `s` is a string and not numeric, but since it is, we can prefix "unknown: " or similar.)

Reply:
sure, seems reasonable